runtime/Makefile

```diff
@@ -12,6 +12,7 @@ BUILD_TYPE?=Release
 ENABLE_WARNINGS?=ON
 ENABLE_OPENMP?=ON
 ENABLE_LIGHTNING_KOKKOS?=OFF
+ENABLE_OPENQASM?=OFF
```

Comment:
Should we not build the device by default? I don't know which would be better though🤔

Reply:
This is a standalone device and let's keep this OFF by default until we implement the end-to-end support and finalize the UI.